docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -125,72 +129,12 @@ The path to the Pantheon data directory. The default is the `/build/distribution
 ### dev-mode
 
 !!!important
-    This option is deprecated in favor of the new `--network` option.
-    It will be completely removed in the 0.9 release.
-
-```bash tab="Syntax"
---dev-mode
-```
-
-```bash tab="Example Configuration File"
-dev-mode=true
-```
-  
-Set this option to `true` to run in development mode. 
-For example, specify this option to perform CPU mining more easily in a private test network. 
-In development mode, a custom genesis configuration specifies the chain ID. 
-When using this option, also set the [`--network-id`](#network-id) option to the network you use for development.
-Default is `false`.
-  
-  
-!!!note
-    The [`--dev-mode`](#dev-mode) option overrides the [`--genesis`](#genesis) option. If both are specified, the development mode configuration is used.  
-
-
-### genesis-file
-
-```bash tab="Syntax"
---genesis-file=<FILE>
-```
-
-```bash tab="Example Command Line"
---genesis-file=/home/me/me_node/customGenesisFile.json
-```
-
-```bash tab="Example Configuration File"
-genesis-file="/home/me/me_node/customGenesisFile.json"
-```
-
-The path to the genesis file. The default is the embedded genesis file for the Ethereum mainnet. 
-When using this option, it is recommended to also set the [`--network-id`](#network-id) option.
-
-!!!note
-    This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#custom-genesis-file). 
-
-!!!note
-    The [`--genesis`](#genesis) option is overridden by the [`--dev-mode`](#dev-mode) option. 
-    If both are specified, the specified genesis file is ignored and the development mode configuration used. 
-
+    This option was removed in favor of the new [`--network`](#network) option.
```

Comment:
Similar to above.  I think it's going to be clearest to include a summary of the command line changes in the release notes and have the 0.9 doc, just doc them as they are in 0.9.